deltachat-ffi/deltachat.h

```diff
@@ -1053,6 +1053,17 @@ int dc_send_webxdc_status_update (dc_context_t* context, uint32_t msg_id, const
  */
 char* dc_get_webxdc_status_updates (dc_context_t* context, uint32_t msg_id, uint32_t serial);
 
+
+/**
+ * List all webxdc that have updates in the queue
+ * @param context The context object.
+ * @param msg_id The ID of the message with the webxdc instance.
+ * @return 1=contact ID is member of chat ID, 0=contact is not in chat
```

Comment:
Being new to this PR, I don't understand what this function does. Does it list all webxdcs with updates? If so, how does it return this using an int? Or does it return whether some contact ID is a member of a chat? If so, which contact?

Reply:
ah my bad, did't update the description correctly